source/administration/devops-command-center.rst

```diff
@@ -0,0 +1,250 @@
+
+Editing checklists
+~~~~~~~~~~~~~~~~~~~
+
+You can add steps and edit checklist items during an active incident. You can also edit checklist items in playbooks that are not being used in active incidents. 
+
+To edit a checklist, choose **Edit**.
+
+- Select the text field to edit the item details. 
+- Hover over an input box and select the **X** to remove the checklist item.
+- Drag the hamburger menu to rearrange the checklist items.
+
+Choose **Done** to save the changes.
+
+Ending incidents
+~~~~~~~~~~~~~~~~
+
+There are two ways to end an incident:
+
+- Using slash command */incident end* from within the incident channel.
+- Using the **End Incident** button in the RHS panel while in the incident channel.
+
+The incident will become inactive, be removed from the list of active incidents, and the associated channel will be archived.
+
+Administrator's Guide
+^^^^^^^^^^^^^^^^^^^^^^
+
+Permissions
+~~~~~~~~~~~~~~~~~~~~~
+
+Incidents and playbooks are associated with teams in Mattermost. Participants can be added by inviting them to the incident's channel. 
+
+To view incidents associated with your Mattermost team, select **! Incidents** from the plugin menu, or **!** in the channel header to open the RHS menu. If there are no active incidents, you can start one or you can view details of incidents that have ended.
+
+Incidents Backstage
+~~~~~~~~~~~~~~~~~~~~
+
+To open the Incidents backstage, which lists all incidents associated with the current team, select **Incidents & Playbooks** from the Main Menu. All incidents for the current team are listed for review with the following details for each incident:
+
+- Name
+- Status (**Ongoing** or **Ended**)
+- Start Date
+- End Date, if ended, otherwise **--**
+- Commander
+
+Viewing incident details
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To view details of active incidents, select **! Incidents** from the plugin menu, or **!** in the channel header to open the RHS panel.
+
+Listed incidents include the following information:
+
+- The current commander’s profile picture and username
+- Zero or more checklist items
+- The channels associated with the incident
+- A button to end the incident
+
+If the incident is active, but in a private channel, you won't be able to see the incident channel name or select **End Incident**. However, the commander name and incident checklist are displayed. If you're a participant in an incident channel, you can modify incident details from within that channel.
+
+You can view all incidents in the incident Backstage via **Main Menu > Incidents & Playbooks** and then select an incident to view the following details:
+
+- Incident name
+- A link icon to open the corresponding incident channel
+- Status (**Ongoing** or **Ended**)
+- The commander, including profile picture and username
+- A prompt to `export the incident channel <#exporting-channels>`_
+- The total duration
+- The number of members involved at the time the incident started
```

Comment:
@justinegeffen what I meant here is that the number you see in the Incident Details is the current number of members in the incident channel. So if you open the details the following day and someone has left the channel in the meantime, you will see that the number has decreased. Sorry for the misunderstanding, I re-read my comment and it was a bit confusing! As a further clarification, I'd say something like the following:
```suggestion
- The number of members currently involved in the incident
```